src/mumble/ListenerVolumeSlider.h

```diff
@@ -20,12 +23,20 @@ class ListenerVolumeSlider : public VolumeSliderWidgetAction {
 	void setListenedChannel(const Channel &channel);
 
 private:
-	/// The channel of the listener proxy this dialog is operating on
+	/// The channel of the listener proxy this widget is operating on
 	const Channel *m_channel;
 
+	QTimer m_sendTimer;
+	QTimer m_resetTimer;
+	unsigned int m_currentTimer;
```

Comment:
The name makes it sound like this variable would also hold a timer object. Maybe
```suggestion
	unsigned int m_currentSendDelay;
```
would make things more clear?